e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -344,6 +344,100 @@ test('Example Imagery in Display layout', async ({ page }) => {
+            page.waitForNavigation(/*{ url: 'http://localhost:8080/#/browse/mine/7f960280-ff54-44b0-8123-b7353e72ca5b?tc.mode=fixed&tc.startBound=1653059038449&tc.endBound=1653060838449&tc.timeSystem=utc&view=layout.view' }*/),
+            page.locator('text=OK').click()
+        ]);
+
+        // Click text=Snapshot Save and Finish Editing Save and Continue Editing >> button >> nth=1
+        await page.locator('text=Snapshot Save and Finish Editing Save and Continue Editing >> button').nth(1).click();
+
+        // Click text=Save and Finish Editing
+        await page.locator('text=Save and Finish Editing').click();
+
+        // Click button:has-text("Create")
+        await page.locator('button:has-text("Create")').click();
+
+        // Click li:has-text("Example Imagery")
+        await page.locator('li:has-text("Example Imagery")').click();
+
+        // Click text=Properties Title Notes Images url list (comma separated) Image load delay (milli >> input[type="text"]
+        await page.locator('text=Properties Title Notes Images url list (comma separated) Image load delay (milli >> input[type="text"]').click();
+
+        // Fill text=Properties Title Notes Images url list (comma separated) Image load delay (milli >> input[type="text"]
+        await page.locator('text=Properties Title Notes Images url list (comma separated) Image load delay (milli >> input[type="text"]').fill('Thumbnail Example Imagery');
+
+        // Click text=OK
+        await Promise.all([
+            page.waitForNavigation(/*{ url: 'http://localhost:8080/#/browse/mine/7f960280-ff54-44b0-8123-b7353e72ca5b/0f14df02-aea0-4ca5-8605-929a5706da0c?tc.mode=fixed&tc.startBound=1653059038449&tc.endBound=1653060838449&tc.timeSystem=utc&view=example.imagery' }*/),
+            page.locator('text=OK').click()
+        ]);
+
+        // Click text=Thumbnail Example Imagery Imagery Layout Snapshot >> button >> nth=0
+        await Promise.all([
+            page.waitForNavigation(/*{ url: 'http://localhost:8080/#/browse/mine/7f960280-ff54-44b0-8123-b7353e72ca5b?tc.mode=fixed&tc.startBound=1653059038449&tc.endBound=1653060838449&tc.timeSystem=utc&view=layout.view' }*/),
+            page.locator('text=Thumbnail Example Imagery Imagery Layout Snapshot >> button').first().click()
+        ]);
+
+        // Edit mode
+        await page.locator('text=Thumbnail Display Layout Snapshot >> button').nth(3).click();
+
+        // Click on example imagery to expose toolbar
+        await page.locator('text=Thumbnail Example Imagery Snapshot Large View').click();
+
+        // expect thumbnails not be visible when first added
+        await expect.soft(thumbsWrapperLocator.isHidden()).toBeTruthy();
+
+        // Resize the example imagery vertically to change the thumbnail visibility
+        // Click #mct-input-id-103
+        await page.locator('#mct-input-id-103').click();
+
+        // Fill #mct-input-id-103
+        await page.locator('#mct-input-id-103').fill('50');
```

Comment:
can you explain the significance of these two fill values in a comment?

Reply:
Needed a reliable way to resize the example imagery so that it would trigger the 3 separate conditions for the thumbnails (hidden, small thumbnails, regular thumbnails). Specifically I am setting the height to 50px so it exceeds the threshold for small thumbnails.

Reply:
A mouse drag didn't seem to be reliable when using the playwright codegen.

Reply:
@michaelrogers could you update the comment in the testcode with that info? That's a great solution, but may get lost to other contributors